charts/fluid-dataloader/alluxio/templates/cronjob.yaml

```diff
@@ -61,26 +61,7 @@ spec:
           {{- end }}
```

Comment:
Please update the chart version and change log.

Reply:
Done